pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -597,12 +650,21 @@ func (o *MustGatherOptions) getGatherContainerLogs(pod *corev1.Pod) error {
 	}
 }
 
-func newPrefixWriter(out io.Writer, prefix string) io.Writer {
+func (o *MustGatherOptions) newPrefixWriter(out io.Writer, prefix string, ignoreFileWriter bool) io.Writer {
```

Comment:
In which circumstance we should pass `ignoreFileWriter` to true?

Reply:
E.g. when logs are printed for individual must-gather running container logs.